src/components/disable-container.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <div class="disable-container">
+    <div v-show="disabled" class="disable-container-overlay" tabindex="0" v-tooltip.sr-only></div>
```

Comment:
I forget why we have `tabindex` here. Why is it helpful to make the overlay focusable?

Reply:
without `tabindex` we can't focus it using only keyboard (pressing tab key to focus different elements on the page)

Reply:
I'm more wondering, why is it important to be able to focus the overlay using the keyboard? The overlay doesn't have content, and it doesn't have an effect when clicked. Does allowing the user to focus the overlay help them better navigate the page?

By the way, I don't think this comment needs to block merge if it'd be helpful to merge the PR sooner. Even if we end up concluding that `tabindex="0"` isn't needed, we can always remove it in a follow-up PR.

Reply:
I find it helpful to know what overlay is all about when I use only keyboard. With mouse hover, one can quickly see the tooltip to know the reason but if someone is just using keyboard then without tabindex there is no way to see the tooltip.